block/block.go

```diff
@@ -412,3 +412,40 @@ func (b *Block) GetReceiverDID() string {
 	}
 	return tt.(string)
 }
+
+func (b *Block) GetTid() string {
+	tt, ok := b.bm[TCTIDKey]
+	if !ok {
+		return ""
```

Comment:
Since saving empty tid is not okay, is it better to throw an error here so that we know it from the console?